package.json

```diff
@@ -7,6 +7,7 @@
     "analyze": "webpack-bundle-analyzer build/production/generated/stats.json",
     "build": "node script/build.js",
     "build-analyze": "NODE_ENV=production npm run build -- --buildtype production --analyzer; npm run analyze",
+    "generate-link-csv:brand-consolidation": "node script/build.js --brand-consolidation-enabled && node script/csv-path-builder",
```

Comment:
If we think this will always run alongside a build, writing it as a metalsmith plugin would simplify the script some.

Reply:
This with be a one off things. Only when we need it and not every time we build. @jkassemi told me that he wanted it to be in a separate script so we can run it separately.